lib/livebook/hubs/dockerfile.ex

```diff
@@ -157,6 +157,45 @@ defmodule Livebook.Hubs.Dockerfile do
     |> Enum.join("\n")
   end
 
+  @doc """
+  Builds Dockerfile definition for Livebook Agent app deployment.
+  """
+  @spec build_agent_dockerfile(config(), Hubs.Provider.t()) :: String.t()
+  def build_agent_dockerfile(config, hub) do
+    base_image = Enum.find(Livebook.Config.docker_images(), &(&1.tag == config.docker_tag))
+
+    image = """
+    FROM ghcr.io/livebook-dev/livebook:#{base_image.tag}
+    """
+
+    image_envs = format_envs(base_image.env)
+
+    hub_config = """
+    ARG AGENT_KEY
+    ARG AGENT_NAME
+
+    # Teams Hub configuration for Livebook Agent deployment
+    ENV LIVEBOOK_AGENT_NAME ${AGENT_NAME}
+    ENV LIVEBOOK_TEAMS_KEY "#{hub.teams_key}"
+    ENV LIVEBOOK_TEAMS_AUTH "online:#{hub.hub_name}:#{hub.org_id}:#{hub.org_key_id}:${AGENT_KEY}"
+    """
+
+    apps_config = """
+    # Apps configuration
+    ENV LIVEBOOK_APPS_PATH "/apps"
+    ENV LIVEBOOK_APPS_PATH_HUB_ID "#{hub.id}"
+    """
```

Comment:
@josevalim do we want to keep these for the agent (since we don't copy any notebooks there anyway)?

Reply:
No, we don't need to keep those.